l10n/configs/vendor.toml

```diff
@@ -0,0 +1,27 @@
+# All content handled by the vendor or Mozilla staff should
+# be covered by the [[paths]] in this file, with the relevant locales.
+
+basepath = ".."
+
+locales = [
+    "de",
+    "fr",
+]
+
+## Examples:
+
+# [[paths]]
+#     reference = "en/firefox/*.ftl"
+#     l10n = "{locale}/firefox/*.ftl"
+# [[paths]]
+#     reference = "en/firefox/*.ftl"
+#     l10n = "{locale}/firefox/*.ftl"
+#     locales = [
+#         "de",
+#         "fr",
+#         "ru",
+#     ]
+
+[[paths]]
+    reference = "en/mozorg/mission.ftl"
```

Comment:
I don't think the mission page is going to be handled by the vendor.

Reply:
Nevermind. I missed your comment in the description.

Reply:
If `de` and `fr` are to be handled by Mozilla staff as opposed to community, then would we need to list out every `.ftl` file here explicitly, or could we just say "all .ftl files that map to either de or fr locales"?

Also if both, vendor and staff are covered in this config file, do we need a way to specify who's responsible for what?

Reply:
From our discussion in Slack the other day, it sounds like Marketing are only responsible for a sub-set of pages (which are still tbd) for both German and French, so we may not be able to create such general rules as I queried about above.